setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
+
+setup(
+    name="simpleocr",
+    packages=["simpleocr"],
+    version="1.0.0",
+    description="A library for simple OCR in Python using OpenCV",
+    author="The simple-ocr-opencv authors",
+    url="https://www.github.com/goncalopp/simple-ocr-opencv",
+    download_url="https://www.github.com/goncalopp/simple-ocr-opencv/archive/1.0.0.tar.gz",
+    keywords=["OCR", "OpenCV"],
+    license="AGPL",
+    classifiers=["Programming Language :: Python :: 2.7",
+                 "Programming Language :: Python :: 3",
+                 "License :: OSI Approved :: GNU Affero General Public License v2 or later (AGPLv2+)"],
+    include_package_data=True,
+    install_requires=["opencv-python"]
```

Comment:
I'd rather not introduce a dependency on a self-declared "unofficial" copy of OpenCV, *especially* a precompiled one - this can easily create a big security problem, so I'm not comfortable making that choice for the user.

My suggestion would be that, at least for linux, we proceed with installation without installing cv, and `try: import cv2` on `__init__.py`, printing a nice error message if it's not there explaining the several possible ways to get it.

I'm guessing for windows this might be less of a issue, since the average user downloads and runs untrusted binaries all the time? I'll let you make a judgement call here.

I think `six`, and `pillow` should go here, right?